packages/node_modules/@webex/plugin-meetings/test/integration/spec/testUtils.js

```diff
@@ -134,11 +134,22 @@ const waitForEvents = (scopeEvents) => {
               }
             }
 
-            if (result.length === scopeEvents.length) {
+            if (obj.match) {
```

Comment:
Why do the `if` statement? You do the exact same 3 methods either way.

Reply:
I left it for backwards compatibility. The previous test cases expected a specific value to be returned and was using `.once` above instead of `.on`. It's possible we can get various member updates (or other meeting updates) in the time the test is running. The idea was to go back at some point and move everything to use `match` since it's a little cleaner in the test itself.